### PR TITLE
chore: Python 3.10

### PR DESCRIPTION
**Pull request by frankh**
## Problem

Performance gains go brrr

## Changes

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

## How did you test this code?

- [x] CI
- [x] running locally

### CODE REVIEW

File: runtime.txt [hunk 1]
```diff
@@ -1 +1 @@
-python-3.9.16
+python-3.10.7
```
Comment (tomasfarias) [inline runtime.txt:1]: Any reason why we are not on the latest 3.10 patch? 3.10.10 was released on Feb 8th.

**Review by tomasfarias — APPROVED** [review]
3.10 is a big one, just the better error messages make this worth it. Thank you 🎉

File: runtime.txt (hunk 1 above)
Comment (tomasfarias) [reply runtime.txt:1]: Miscellaneous bugfixes in the changelog from 3.10.7 to 3.10.10 as far as I can tell.

Comment (frankh) [reply runtime.txt:1]: Oops, no reason, just Google lied to me when I searched latest python version 😔

**Comment by hazzadous** [comment]
@frankh 3.11 ? ⏲️

**Pull request merged**